src/elements/content-sidebar/activity-feed/task-new/AvatarGroup.scss

```diff
@@ -0,0 +1,107 @@
+@import '../../../common/variables';
+@import './mixins';
+
+$avatar-spacing: 4px;
```

Comment:
This is being declared as a global variable. Is that intended?

Reply:
Fixed.